java/libphonenumber/test/com/google/i18n/phonenumbers/CountryCodeToRegionCodeMapForTesting.java

```diff
@@ -31,10 +31,10 @@
   // countries sharing a calling code, such as the NANPA countries, the one
   // indicated with "isMainCountryForCode" in the metadata should be first.
   static Map<Integer, List<String>> getCountryCodeToRegionCodeMap() {
-    // The capacity is set to 33 as there are 25 different entries,
+    // The capacity is set to 36 as there are 27 different entries,
```

Comment:
Weird that this increased by 3. I see only 2 new regions: TA and +882 (just wondering)